azurelinuxagent/distro/suse/rdma.py

```diff
@@ -0,0 +1,72 @@
+# Microsoft Azure Linux Agent
+#
+# Copyright 2014 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.4+ and Openssl 1.0+
+#
+
+import glob
+import os
+import azurelinuxagent.logger as logger
+from azurelinuxagent.distro.default.rdma import RDMAHandler
+
+
+class SUSERDMAHandler(RDMAHandler):
+    def install_driver(self):
+        """Install the appropropriate driver package for the RDMA firmware"""
+
+        fw_version = self._RDMAHandler__get_rdma_version()
+        zypper_install = 'zypper -n in'
+        zypper_search = 'zypper se -s'
+        package_name = 'msft-rdma-kmp-default'
+        cmd = '%s %s' % (zypper_search, package_name)
+        repo_package_info = os.popen(cmd).readlines()
```

Comment:
Can we use `shellutil.run_get_output` please? That would allow us to have the external exec logic in a central place for easier refactoring in the future.